src/activity.js

```diff
@@ -0,0 +1,70 @@
+const getItemFromLocalStorage = () => {
+  let data = JSON.parse(localStorage.getItem('tasks'));
+  if (!data) {
+    data = [];
+  }
+  return data;
+};
+
+const saveItemToLocalStorage = (tasks) => {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+const createTask = () => {
+  const tasks = getItemFromLocalStorage();
+  const newTaskDescription = document.getElementById('to-do-input');
+  const newTask = {
+    description: newTaskDescription.value,
+    completed: false,
+    index: tasks.length + 1,
+  };
+  newTaskDescription.value = '';
+  tasks.push(newTask);
+
+  saveItemToLocalStorage(tasks);
+};
+
+const updateTask = (taskId, el) => {
+  const tasks = getItemFromLocalStorage();
+  const task = tasks.find((task) => task.id === parseInt(taskId, 10));
+  if (el.hasAttribute('content-editatable')) {
+    task.description = el.textContent;
+  } else {
+    const span = el.nextElementSimbling;
+    const parent = el.closest('input');
+    task.isCompleted = !task.isCompleted;
```

Comment:
- [ ] Even though you have a `updateTask` function which helps you edit tasks, the edit feature isn't working properly as I am unable to edit any task. Please implement this requirement 🙏